src/ashley/templates/board_base.html

```diff
@@ -7,25 +7,37 @@
 {% endblock css %}
 
 {% block body %}
-{% block header %}{{ block.super }}{% endblock header %}
-<div class="my-0 px-2 container-fluid" id="main_container">
-  <div class="row">
-    <div class="col-12">
-      {% block breadcrumb %}{% include "partials/breadcrumb.html" %}{% endblock breadcrumb %}
-    </div>
-  </div>
-  <div class="row">
-    <div class="col-12">
-      <br />
-      {% block messages %}{% include "partials/messages.html" %}{% endblock messages %}
+    {% block header %}
+    {% endblock header %}
+    <div class="my-0 px-2 container-fluid" id="main_container">
+        <div class="row">
+            <div class="col-12">
+                {% block breadcrumb %}{% include "partials/breadcrumb.html" %}{% endblock breadcrumb %}
+            </div>
+        </div>
+
+        {% block search %}
+        <div class="row">
+            <form class="form-inline my-2 ml-3 my-lg-0" role="search" action="{% url 'forum_search:search' %}">
+                <input class="form-control form-control-sm mr-sm-2" type="search" placeholder="Search" aria-label="Search" name="q">
```

Comment:
What do you think about adding a `required` attribute to the search field to prevent to submit the form with an empty query param?

Reply:
I think it's a very good suggestion! :pray: